### PR TITLE
Add more logs on write path

### PR DESCRIPTION
**Pull request by czs007**
Signed-off-by: zhenshan.cao <zhenshan.cao@zilliz.com>

### CODE REVIEW

File: internal/datanode/data_node.go [hunk 1]
```diff
@@ -303,28 +307,37 @@ func (node *DataNode) handleWatchInfo(key string, data []byte) {
...
+	log.Debug("DataNode handleWatchInfo Unmarshal success")
 	if watchInfo.State == datapb.ChannelWatchState_Complete {
+		log.Warn("DataNode handleWatchInfo State is already ChannelWatchState_Complete")
 		return
 	}
 	if watchInfo.Vchan == nil {
 		log.Warn("found ChannelWatchInfo with nil VChannelInfo", zap.String("key", key))
 		return
 	}
+	log.Warn("DataNode handleWatchInfo try to NewDataSyncService", zap.String("key", key))
```
Comment (sunby) [inline internal/datanode/data_node.go:319]: use Debug level?

File: internal/datanode/data_node.go [hunk 2]
```diff
@@ -303,28 +307,37 @@ func (node *DataNode) handleWatchInfo(key string, data []byte) {
...
+
 	watchInfo.State = datapb.ChannelWatchState_Complete
 	v, err := proto.Marshal(&watchInfo)
 	if err != nil {
-		log.Warn("fail to Marshal watchInfo", zap.String("key", key), zap.Error(err))
+		log.Warn("DataNode handleWatchInfo fail to Marshal watchInfo", zap.String("key", key), zap.Error(err))
 		return
 	}
 	k := path.Join(Params.DataNodeCfg.ChannelWatchSubPath, fmt.Sprintf("%d", node.NodeID), watchInfo.GetVchan().GetChannelName())
+	log.Warn("DataNode handleWatchInfo try to Save", zap.String("key", key),
```
Comment (sunby) [inline internal/datanode/data_node.go:334]: please make it more readable

Comment (sunby) [reply internal/datanode/data_node.go:334]: comment above is also required for other log prints

File: internal/datacoord/channel_manager.go [hunk 3]
```diff
@@ -262,7 +262,15 @@ func (c *ChannelManager) Watch(ch *channel) error {
...
-	return c.store.Update(updates)
+	err := c.store.Update(updates)
+	if err != nil {
+		log.Error("ChannelManager RWChannelStore update failed", zap.Int64("collectionID", ch.CollectionID),
+			zap.String("channelName", ch.Name), zap.Error(err))
+		return err
+	}
+	log.Debug("ChannelManager RWChannelStore update success", zap.Int64("collectionID", ch.CollectionID),
+		zap.String("channelName", ch.Name))
+	return nil
```
Comment (sunby) [inline internal/datacoord/channel_manager.go:273]: Dont log in this method, please return the error directly and  handle it at the top of the callstack.

File: internal/datanode/data_node.go [hunk 4]
```diff
@@ -303,28 +307,37 @@ func (node *DataNode) handleWatchInfo(key string, data []byte) {
...
 		log.Warn("found ChannelWatchInfo with nil VChannelInfo", zap.String("key", key))
 		return
 	}
+	log.Warn("DataNode handleWatchInfo try to NewDataSyncService", zap.String("key", key))
 	err = node.NewDataSyncService(watchInfo.Vchan)
 	if err != nil {
 		log.Warn("fail to create DataSyncService", zap.String("key", key), zap.Error(err))
 		return
 	}
+	log.Warn("DataNode handleWatchInfo NewDataSyncService success", zap.String("key", key))
```
Comment (sunby) [inline internal/datanode/data_node.go:325]: Debug level?

File: internal/datanode/data_sync_service.go [hunk 5]
```diff
@@ -115,16 +115,19 @@ func newParallelConfig() parallelConfig {
 // start starts the flowgraph in datasyncservice
 func (dsService *dataSyncService) start() {
 	if dsService.fg != nil {
-		log.Debug("Data Sync Service starting flowgraph")
+		log.Debug("dataSyncService starting flowgraph", zap.Int64("collectionID", dsService.collectionID),
```
Comment (sunby) [inline internal/datanode/data_sync_service.go:118]: There's no need to write log like 'dataSyncService'.  IMHO, 'data sync service' or 'datasync service' is better.

File: internal/datanode/data_node.go (hunk 1 above)
Comment (czs007) [reply internal/datanode/data_node.go:319]: ok. modify in next pr.

File: internal/datanode/data_node.go (hunk 4 above)
Comment (czs007) [reply internal/datanode/data_node.go:325]: ok. modify in next pr.

File: internal/datacoord/channel_manager.go (hunk 3 above)
Comment (czs007) [reply internal/datacoord/channel_manager.go:273]: ok. modify in next pr.

File: internal/datanode/data_sync_service.go (hunk 5 above)
Comment (czs007) [reply internal/datanode/data_sync_service.go:118]: ok. modify in next pr.

**Pull request merged**